console/src/components/Services/ApiExplorer/Analyzer/AnalyzeText.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { MouseEvent } from 'react';
+
+const AnalyzeText: React.FC<{
+  data: Array<object>;
+  activeNode: number;
+  onClick: (e: MouseEvent) => void;
+}> = ({ data, activeNode, onClick }) => (
+  <>
+    {data.map((analysis: any, i: number) => {
```

Comment:
We can skip typings. It will be inferred from the `data` type.
```suggestion
    {data.map((analysis, i) => {
```